src/components/tutorial-meta/components/interactive-lab-button/index.tsx

```diff
@@ -22,6 +22,8 @@ export default function InteractiveLabButton() {
 			onClick={() => ctx.setActive(!ctx.active)}
 			icon={<IconTerminalScreen16 />}
 			data-heap-track="interactive-lab-button"
+			color="secondary"
+			size="small"
```

Comment:
In the designs, this spans the entire width at <tablet sizes, but after looking at it _in situ_ we decided to have it wrap in the same way as on desktop. cc @braydenlove for the final call there though.